agent/engine/docker_task_engine.go

```diff
@@ -474,7 +474,10 @@ func (engine *DockerTaskEngine) GetTaskByArn(arn string) (*api.Task, bool) {
 
 func (engine *DockerTaskEngine) pullContainer(task *api.Task, container *api.Container) DockerContainerMetadata {
 	pullStart := time.Now()
-	defer seelog.Infof("Finished pulling container %v. Lock acquisition and pull took %v.", container, time.Since(pullStart).String())
+	defer func(startTime time.Time) {
+		seelog.Infof("Finished pulling container %v. Lock acquisition and pull took %v.", container, time.Since(startTime).String())
```

Comment:
If the time calculated includes lock acquisition, I'd prefer that we were explicit.  Ideally, I'd like a separate time for the actual pull vs. the pull-plus-lock-acquisition (or maybe logging lock acquisition time alone).